xbmc/win32/AppxManifest.xml.in

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Package xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10" xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10" xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities">
+  <Identity Name="@APP_NAME@" ProcessorArchitecture="x86" Publisher="CN=@COMPANY_NAME@" Version="@VERSION_NUMBER@" />
+  <Properties>
+    <DisplayName>@APP_NAME@</DisplayName>
+    <PublisherDisplayName>@COMPANY_NAME@</PublisherDisplayName>
+    <Description>@APP_NAME@ @APP_VERSION@</Description>
+    <Logo>media\icon120x120.png</Logo>
+  </Properties>
+  <Resources>
+    <Resource Language="en-us" />
+  </Resources>
+  <Dependencies>
+    <TargetDeviceFamily Name="Windows.Desktop" MinVersion="10.0.14342.0" MaxVersionTested="10.0.14379.0" />
```

Comment:
No it can't run on current version, but it's just one extra installer so doesn't block regular use. @afedchin rumors have it the release is August 2nd so you're soon able to try it out without preview version